contrib/test_jenkins.sh

```diff
@@ -589,27 +594,32 @@ run_ucx_perftest_mpi() {
 	do
 		if [[ $ucx_dev =~ .*mlx5.* ]]; then
 			opt_transports="-b $ucx_inst_ptest/transports"
+			tls=`awk '{print $3 }' $ucx_inst_ptest/transports | tr '\n' ',' | sed -r 's/,$//; s/mlx5/x/g'`
```

Comment:
does this convert transports batch file to a list?

Reply:
yes